Sofia-Rojas/tarea2/form1.html

```diff
@@ -0,0 +1,162 @@
+<form action="#">
+
+
+    
+    <header>
+         <h1>Pet adoption form</form></h1>
+    </header>
+
+    <Main">
+        <p>
+            In this form you will fill the information needed to complete the adoption process of a domestic animal. The data you will need is:
+        
+            <ol>
+            <li>Personal information</li>
+            <li>Household information</li>
+            <li>A personal reference and their information</li>
+            </ol>
+        </p>
+
+        <fieldset id="1">
+            <legend>Personal information</legend>
+
+                <label for="date">Date </label>
+
+                <input type="date" id=Date name="Date">
+                 
+
+                <section>
+            
+                    <label for="adopter's name">Name of the adopter:</label>
+                    <input type="text" id="adopter's name" name="name" required>
+                    &nbsp;
+
+                    <label for="birthday">Birthday:</label>
+                    <input type="date" id="birthday" name="birthday"
+                    required>
+                    &nbsp;
```

Comment:
@sofia-rojas , use this kind of charactes for spacing is a bad practice , you can enclose the inputs in p tag, li or sections